src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/CreateReferralController.java

```diff
@@ -181,13 +181,18 @@ public ResponseEntity<CCDCallbackResponse> aboutToSubmitReferralDetails(
                 String.format("%s %s", userDetails.getFirstName(), userDetails.getLastName()),
                 this.documentManagementService.addDocumentToDocumentField(documentInfo));
 
-        String caseLink = emailService.getExuiCaseLink(caseDetails.getCaseId());
+        String caseLink = null;
+        if (StringUtils.isEmpty(caseData.getMultipleReference())) {
+            caseLink = emailService.getExuiCaseLink(caseDetails.getCaseId());
+        }
 
-        if (StringUtils.isNotEmpty(caseData.getReferentEmail())) {
+        if (StringUtils.isNotEmpty(
```

Comment:
We already have an if to cover the other branch above. This could just be an `else`

Reply:
Refactored